client/src/hooks/index.js

```diff
@@ -0,0 +1,4 @@
+import useInput from './useInput';
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
Why?

Reply:
It's telling me to use default export instead of named export. I'm currently exporting only one thing that's why it suggests using default import but eventually, I wanna export all my custom hooks from this file so I'm using named export. Will remove this line when I add more hooks